app/assets/stylesheets/partials/_member_note.scss

```diff
@@ -1,11 +1,8 @@
 #note  {
-  margin-bottom: 1em;
-  padding-bottom: 1em;
-
-  border-bottom: 1px solid #ddd;
-
-  &:last-child {
-    border-bottom: none;
+  & + & {
+    margin-top: 1em;
+    padding-top: 1em;
+    border-top: 1px solid #ddd;
```

Comment:
This CSS change allows us to show stacked notes as previously but it will fix a small issue with putting the notes HTML inside the tooltip, where  the tooltip notch (that triangle) was the last element so we were getting an extra border without any need. This makes it a bit more specific (i.e. only a note followed by another note will have this applied).

Reply:
Does this apply to the new view?

Reply:
You are right; since the view changed, this is probably unnecessary now.